_includes/main_nav_tr.html

```diff
@@ -1,6 +1,13 @@
  <nav class="navbar navbar-expand-lg navbar-light bg-light fixed-top pl-md-5 pr-md-5 pl-sm-3 pr-0">
 
-    <a class="navbar-brand" href="{{ site.url }}/{{ page.lang }}/"><img src="{{ site.url }}/images/bisq-logo.svg" height="30"/></a>
+   <div>
+     <a class="navbar-brand" href="{{ site.url }}/{{ page.lang }}/"><img src="{{ site.url }}/images/bisq-logo.svg" height="30"/></a>
+
+     <!-- Outdated translation alert tool tag-->
+     {% if page.outdated_translation %}
+       <span class="navbar-text" style="color: orange;">Outdated translation</span>
```

Comment:
I suggest to add a link toTransifex, so that potential contributors know where to go if they want to help translate the page. The message could be something like `Outdated translation - [Help us to improve it!](https://www.transifex.com/bisq/bisq-website/)`.

Would be also cool to have the phrase translatable imho, so that when somebody is using another language the warning will be in their language.

Reply:
@erciccione what do you mean by message?

Reply:
The content of the warning (that now is only `Outdated translation`)

Reply:
Well, for that I think we'll have to move it from the navbar. There's not enough space there as it is.

Reply:
I don't think the navbar is the best place for that warning. I would put it in a dedicated snippet under the navbar. That way would be easier to see for the user, we would have more space so we can be more descriptive and styling form mobile would be easier to manage.

Reply:
@erciccione @m52go I've moved it from the nav bar to the page body. Maybe we can work more on the text to display something more inviting and to make sure they check out the English version.
![outdatedtool](https://user-images.githubusercontent.com/44140017/64205787-b5aae880-ce90-11e9-9b9e-173bac191ffe.png)

Reply:
@erciccione how about this, following the standard warning sign palette. Orange text on green background doesn't stand out that much.
![outdated2](https://user-images.githubusercontent.com/44140017/64241528-d6f4ee80-cefb-11e9-91a7-7874cf5c9671.png)

Reply:
The black text however doesn't imply that there's an hyperlink to be clicked.

Reply:
I'm not great with colours so, keep that in mind :P, but i personally think this solution is better than simple text. What about having the hyperlink underlined? or underlined on hover?

Reply:
@ericcione now it looks like this
![https://media.giphy.com/media/gjfbOdyAwekD4JgoFI/giphy.gif](https://media.giphy.com/media/gjfbOdyAwekD4JgoFI/giphy.gif)

Reply:
I guess could be fine. Better wait @m52go's input on this